include/ultra64/gbi.h

```diff
@@ -5134,6 +5136,25 @@ _DW({                                                   \
 #define gDPNoOpCloseDisp(pkt, file, line)   gDma1p(pkt, G_NOOP, file, line, 8)
 #define gDPNoOpTag3(pkt, type, data, n)     gDma1p(pkt, G_NOOP, data, n, type)
 
+#else
+
+#define gDPNoOp(pkt)
+#define gsDPNoOp()
```

Comment:
not sure the static dlist commands `#define gsDPNoOp()` would work like this, for example

```c
Gfx dl[] = {
  gsDPNoOp(),
  gsSPEndDisplayList(),
};
```

would expand to invalid C

(pseudo-expansion:)

```c
Gfx dl[] = {
  ,
  { _SHIFTL(G_ENDDL, 24, 8), 0 },
};
```

not sure what to do about it (note in practice afaik there is no `gsDPNoOp*` anywhere)

Reply:
Good point, I've restored gsDPNoOp and friends, which I think makes things a bit more organized too